lib/bios/menus.robot

```diff
@@ -236,6 +238,7 @@ Press Key N Times And Enter
     [Arguments]    ${n}    ${key}
     Press Key N Times    ${n}    ${key}
     Press Enter
+    Sleep    2s
```

Comment:
@miczyg1 This commit https://github.com/Dasharo/open-source-firmware-validation/pull/134/commits/43fc5db7c1d31d9a49cfb4d39898b4bc974021a6 says about adding interval when saving changes, but in fact it is added for each movement. The additional 2s makes the movement much slower. Was it also necessary?